### DISCUSSION TITLE
Change order of Potions

### DISCUSSION THREAD

**Discussion by Wolve1** [HgAlexx/Buffet#34] [Q&A] [answered]
I am on the latest beta-release .81 for TBC

"Mana Potion Injector" is not in bests.manaPotion included, it goes to heathstone even tough I have the injector in my bags

Also, is there a way to change the order of bests.manaPotion? I personally like to consume my "Auchenai Mana Potions" before my "Super Mana Potions" which seems to have a higher priority

**Comment by HgAlexx** [comment, marked as answer]
Hello,

The mana injector issue has been fixed but I haven't got the time to do a proper release :(

I've been working on few other issues which are taking me longer than expected, in addition of not having much time to work on Buffet lately :(

In the mean time, try using the last version from github directly:
https://github.com/HgAlexx/Buffet/archive/refs/heads/master.zip

Regards,

**Reply by Wolve1** [reply]
Do I as the user have the option to change the order of bests.manaPotions?

**Reply by HgAlexx** [reply]
No, the bests variable only show the best of each categories.

However, if a restricted items is allowed and have the same value as the current best, it should take priority.

I'm currently working on issue #36 which will give you access to all available items.
